.chloggen/enable-loki-reciver.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'new_component'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: lokireceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable lokireceiver component
```

Comment:
I'm worried this changelog entry will be a little confusing due to how we release Contrib from `-Releases` and not from this repository.  Adding this component to `otelcontribcol` only adds it to to collector created by `make otelcontribcol`, it won't add it to our official contrib distro.  How do you feel about changing this to something like `mark lokireceiver as Alpha`?

Reply:
chose `Add loki receiver to the build`. I think it is the most precise variant

Reply:
That section of the contributing doc should be updated in my opinion.  Adding the component to `otelcontribcol` is definitely important, but it doesn't have any meaning outside of dev/test.  The milestone moment with the third component is that it is now in Alpha stability and ready to be used by customers.  By marking the component Alpha we indicate to users building their own distributions of the collector with ocb that this component is ready to be included.  It also means it is ready to be included in the official community builds.